src/components/Book.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Book({ book }) {
```

Comment:
## Convert to ES6 syntax
- [ ] We need to ensure we are using ES6 syntax all throughout the codebase. Kindly convert this `Book` component function to an arrow based function.
```suggestion
export default const Book = ({ book }) => {
```